jest.config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "collectCoverage": false,
+  "coverageReporters": ["lcov", "text"],
+  "collectCoverageFrom": ["**/src/**/*.ts", "**/config/**/*.ts"],
+  "transform": {
+    "^.+\\.tsx?$": "ts-jest"
+  },
+  "testMatch": ["<rootDir>/**/?(*.)spec.ts"],
```

Comment:
![image](https://user-images.githubusercontent.com/10531605/87673162-799c1180-c74a-11ea-89cc-f34909ae8c8f.png)
I tried to run the tests, maybe because using windows, the test match didn't work.
As a suggestion, I could change the expression for this one:
`"testMatch": [
    "**/*.spec.ts"
  ]`